python/lsst/ts/MTAirCompressor/aircompressor_csc.py

```diff
@@ -81,7 +81,7 @@ class MTAirCompressorCsc(salobj.BaseCsc):
     valid_simulation_modes: typing.Sequence[int] = (0, 1)
 
     def __init__(
-        self, index: int, initial_state=salobj.State.STANDBY, simulation_mode: int = 0
+        self, index: int, initial_state=salobj.State.DISABLED, simulation_mode: int = 0
```

Comment:
I think the standard is to begin from `STANDBY`. If you want to begin from `DISABLED`, maybe some explanation will be helpful!